app/jobs/reindex_item_job.rb

```diff
@@ -2,6 +2,6 @@
 
 class ReindexItemJob < ApplicationJob
   def perform(item)
-    item.update_index
+    Hyrax.index_adapter.save(resource: item)
```

Comment:
```suggestion
    Hyrax.query_service.find_all_of_model(model: item.class).each do |resource|
      Hyrax.index_adapter.save(resource: item)
    end
```

Should this turn into the above, based on our convo with Rob? Would this work if there are existing active fedora objects?

Reply:
I have to blow away my env to confirm ^^ checking now though...

Reply:
this was the error i received: 
![image](https://github.com/samvera/hyku/assets/10081604/4147e536-e2c9-46c8-9f91-3335ae1b9509)